src/ffs.cpp

```diff
@@ -105,16 +105,16 @@ namespace libtorrent { namespace aux
 		return num * 32;
 	}
 
-	int clz(span<std::uint32_t const> buf)
+	int count_leading_zeros(span<std::uint32_t const> buf)
 	{
 #if TORRENT_HAS_BUILTIN_CLZ || defined _MSC_VER
-		return aux::clz_hw(buf);
+		return aux::count_leading_zeros_hw(buf);
 #else
-		return aux::clz_sw(buf);
+		return aux::count_leading_zeros_sw(buf);
 #endif
 	}
 
-	int flz_sw(span<std::uint32_t const> buf)
+	int find_last_zero_sw(span<std::uint32_t const> buf)
```

Comment:
sorry for nit-picking. normally when searching, I would expect to search from left to right, but here "last" means right-most, or highest bit-index. I would expect there to be a less ambiguous name. I can't think of any great name right now.